src/ui/equipment.js

```diff
@@ -11,11 +11,19 @@ export class Equipment {
     equipment = {
         WEAPON: {},
     };
+    bg = {
```

Comment:
Can we get a better name for this?

Reply:
bg = background. Do you mean you think it should be background instead or totally different?

Reply:
bgWhenEquipped or something like that

Reply:
equipmentSlotBg?